string label
sensor_msgs/Image img
---
bool result